allennlp/training/trainer.py

```diff
@@ -195,6 +196,47 @@ def on_end(
 TrainerCallback.register("null")(TrainerCallback)
 
 
+@TrainerCallback.register("sanity-checks")
+class SanityCheckCallback(TrainerCallback):
+    """
+    Performs model sanity checks.
+    """
```

Comment:
It would be nice if somewhere in this class it explained that even though we have only one sanity check right now, this is the place to add additional ones.